### PR TITLE
Added prop type helpers

### PR DESCRIPTION
**Pull request by jschr**
```js
// mira.config.js
import { number } from 'mira-kit/prop-types';

export default {
  name: 'My Mira App',
  presentationProperties: {
    duration: number('Duration')
      .min(5)
      .default(10)
      .helperText('Time in seconds.'),
  },
  applicationVariables: {
    'New Presentation': {},
  },
};
```

### CODE REVIEW

File: packages/mira-kit/typings.d.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
+declare module 'mira-kit/prop-types' {
```
Comment (KiaFathi) [inline packages/mira-kit/typings.d.ts]: Typescript!

File: packages/mira-kit/package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "name": "mira-kit",
```
Comment (KiaFathi) [inline packages/mira-kit/package.json]: Would like to see readme updates as part of this PR

File: packages/mira-kit/typings.d.ts (hunk 1 above)
Comment (jschr) [reply packages/mira-kit/typings.d.ts]: Just for auto-complete goodness

File: packages/mira-kit/package.json (hunk 2 above)
Comment (jschr) [reply packages/mira-kit/package.json]: There's more than just presentation properties that need to be updated in the docs. Was planning on re-structuring next week along with examples.

Would a google doc be a better for place for collaboration and comments?

**Pull request merged**